app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -254,6 +273,10 @@ def feedback_status_colour(reference)
       end
     end
 
+    def confidentiality_value
+      reference.confidential ? 'No, this reference is confidential. Do not share it.' : 'Yes, if they request it.'
```

Comment:
Have we considered moving this to a translation?
`t('support_interface.references.confidential_warning', reference.confidential)`

```yaml
support_interface: 
  references:
    confidential_warning:
      true: No, this reference is confidential. Do not share it.
      false: Yes, if they request it.
```

Reply:
I did consider it but I didn't use translations in this case because I followed the existing pattern in this file. 

If we were to put this in translations, it would make sense to move all text in this file to translations for consistency. I personally find it a bit overkill to use translations for text that's only used in one place, especially if there aren't any immediate plans to reuse it elsewhere or support multiple languages.

I know we do use it everywhere in Apply though. What are your thoughts?

Reply:
I feel that when we're adding new text we should aim to add it as a translation. We can worry about moving other text later 👌

Reply:
Thanks for the clarification. I’ll update this to use a translation and leave the rest of the text in the file as it is for now.

🤔 I couldn't get your suggestion to quite work for some reason so I've gone with the more basic one [here](https://github.com/DFE-Digital/apply-for-teacher-training/pull/10137/commits/fb8b00ea00d91ce9b0ba4b2125d546c7cd00d4d8): 

```
t("support_interface.references.confidential_warning.#{reference.confidential}")
```